packages/perseus/src/widgets/interactive-graphs/graphs/use-transform.ts

```diff
@@ -1,10 +1,36 @@
-import {useTransformContext, vec} from "mafs";
-
-export const useTransform = (...points: vec.Vector2[]) => {
-    const {viewTransform, userTransform} = useTransformContext();
-    const transformToPx = vec.matrixMult(viewTransform, userTransform);
-    const transformedPoints = points.map((point) =>
-        vec.transform(point, transformToPx),
-    );
-    return transformedPoints;
+import {type Interval, vec} from "mafs";
+
+import useGraphConfig from "../reducer/use-graph-config";
+
+const matrixBuilder = vec.matrixBuilder;
+
+export type GraphDimensions = {
+    range: [Interval, Interval];
+    width: number; // pixels
+    height: number; // pixels
+};
+
+export function vectorToPixel(
+    points: vec.Vector2[],
+    graphState: GraphDimensions,
+    translation: vec.Vector2 = [0, 0],
+) {
+    const {range, width, height} = graphState;
+    const [[xMin, xMax], [yMin, yMax]] = range;
+    const transformToPx = matrixBuilder()
+        .translate(...translation)
+        .scale(width / (xMax - xMin), -height / (yMax - yMin))
+        .get();
+    return points.map((p) => vec.transform(p, transformToPx));
+}
+
+export function pointToPixel(point: vec.Vector2, graphState: GraphDimensions) {
+    const [[xMin], [, yMax]] = graphState.range;
+    const [a, b] = [-xMin, -yMax];
+    return vectorToPixel([point], graphState, [a, b]);
+}
+
+export const useTransformVectorToPixel = (...points: vec.Vector2[]) => {
```

Comment:
same comment about usage of `points` as a parameter name here. 
```suggestion
export const useTransformVectorToPixel = (...vectors: vec.Vector2[]) => {
```

Reply:
Did some additional refactoring in this file to get plural/singular syntax into alignment. Thanks for pointing this out in so many places!